services/code.service.js

```diff
@@ -7,8 +7,7 @@ const path = require('path')
 const sqlite3 = require('sqlite3').verbose()
 const { PYTHON, PROMPTV1, PROMPTV2, PROMPTV3 } = require('../enums/supportedLanguages')
 const logger = require('../loader').helpers.l
-const OpenAI = require('openai')
-const openai = new OpenAI()
+const { openai } = require('../helpers/openaiInstance')
```

Comment:
Has this flow been tested for subjective evaluations? As they use openAI calls to evaluate answers

Reply:
Made the requested changes, and rebuilt the docker container, and the post requests were successfully sent via Postman and have been traced on Langfuse.